### PR TITLE
feature: create project domain

### PR DESCRIPTION
**Pull request by amordzon**
Screenshot 2024-08-08 at 15 08 25
Screenshot 2024-08-08 at 15 13 52
Screenshot 2024-08-08 at 15 14 27

### CODE REVIEW

File: src/main/java/org/example/controllers/ProjectController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    .status(Response.Status.CREATED)
+                    .entity(projectService.createProject(projectRequest))
+                    .build();
+
+        } catch (SQLException e) {
+            return Response.serverError().build();
+
+        }
+//        catch (InvalidException e) {
+//            return Response.status(Response.Status.BAD_REQUEST)
```
Comment (Alicja-Kainos) [inline src/main/java/org/example/controllers/ProjectController.java:40]: is that necessary?

File: src/main/java/org/example/daos/ProjectDao.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    public void updateProject(final int id, final ProjectStatus status)
+            throws SQLException {
+        Connection c = DatabaseConnector.getConnection();
+
+        String updateStatement = "UPDATE Project SET Status = ? WHERE id = ?";
+
+        PreparedStatement st = c.prepareStatement(updateStatement);
+
+        st.setString(1, String.valueOf(status));
```
Comment (Alicja-Kainos) [inline src/main/java/org/example/daos/ProjectDao.java:82]: magic number ;)

File: src/main/java/org/example/controllers/ProjectController.java (hunk 1 above)
Comment (amordzon) [reply src/main/java/org/example/controllers/ProjectController.java:40]: i left it commented so i would remember to implement it, should i remove it?

File: src/main/java/org/example/daos/ProjectDao.java (hunk 2 above)
Comment (amordzon) [reply src/main/java/org/example/daos/ProjectDao.java:82]: i hate magic numbers :((( but ok i will fix that

**Review by Alicja-Kainos — APPROVED** [review]

**Review by PiotrMichalak1 — APPROVED** [review]

**Pull request merged**